universal-login-commons/lib/core/utils/sanitizers/asTransferDetails.ts

```diff
@@ -1,10 +1,9 @@
 import {asObject, asString, Sanitizer} from '@restless/sanitizers';
 import {TransferDetails} from '../../..';
-import {asEthAddress} from '@restless/ethereum';
 import {asGasParameters} from './asGasParameters';
 
 export const asTransferDetails: Sanitizer<TransferDetails> = asObject<TransferDetails>({
-  to: asEthAddress,
+  to: asString,
```

Comment:
what do you think about introduce new sanitizer? (asRecipient)

Reply:
+1, possibly in follow up

Reply:
Let's do it in nex pr. This one is big enough.